database/migrations/021-fix-inventories-and-pets.sql

```diff
@@ -0,0 +1,7 @@
+-- Up
+
+DELETE FROM inventory_slots WHERE slot = 1 AND itemId = 0;
```

Comment:
yavais pas le meme bug avec le slot 2 ?

Reply:
Le premier slot c'est le 0 donc celui là c'est le 2. Et avec le slot 0 aucun soucis

Reply:
ok why not mdr